src/schema/v2/partner.ts

```diff
@@ -35,13 +40,60 @@ const artworksArgs: GraphQLFieldConfigArgumentMap = {
   },
 }
 
+const ArtistPartnerConnection = connectionWithCursorInfo({
+  name: "ArtistPartner",
+  nodeType: ArtistType,
+  edgeFields: partnerArtistFields,
+}).connectionType
+
 export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
   name: "Partner",
   interfaces: [NodeInterface],
   fields: () => {
     return {
       ...SlugAndInternalIDFields,
       cached,
+      artistConnection: {
+        description: "A connection of artists represented by a Partner.",
```

Comment:
The represented status is an edge field (and possible arg), so maybe "A connection of artists at a partner." or something.